src/Microsoft.Azure.WebJobs.Host/Blobs/ParameterizedBlobUrl.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+
+namespace Microsoft.Azure.WebJobs.Host.Blobs
+{
+    internal class ParameterizedBlobUrl : IBindableBlobPath
+    {
+        private readonly BindingTemplate _urlBindingTemplate;
+
+        public ParameterizedBlobUrl(BindingTemplate urlBindingTemplate)
+        {
+            Debug.Assert(urlBindingTemplate != null);
+
+            _urlBindingTemplate = urlBindingTemplate;
+        }
+        public string ContainerNamePattern
+        {
+            get { return _urlBindingTemplate.Pattern + ".getContainer()"; }
```

Comment:
>".getContainer()"; [](start = 55, length = 18)

Can you add a comment like:
"the return value here is just used for logging and does not need to be a valid binding expression"